README.md

```diff
@@ -43,5 +61,75 @@ $EDITOR /path/to/config/config.json
 podman run --name samba  --publish 10445:445 --volume=/path/on/host/to/share:/share:Z --volume=/path/to/config:/etc/samba-container -e SAMBACC_CONFIG=/etc/samba-container/config.json -e SAMBA_CONTAINER_ID=myid  --rm  quay.io/samba.org/samba-server:latest
 ```
 
-The configuration may be used to define custom users or different share
-configurations.
+<!-- TODO: link to advanced docs for samba server -->
+
+## AD DC
+
+The AD DC image defaults to the `samba-dc-container` entrypoint.
+
+In the default configuration, the AD DC container image automatically
+provisions and serves a simple stock domain `DOMAIN1.SINK.TEST`.
+Because the Samba AD DC uses certain file system xattrs this container
+must currently be run with privileges. Example:
+
+```sh
+podman run --rm --privileged  quay.io/samba.org/samba-ad-server:latest
+```
+
+The `samba-dc-container` entrypoint can perform multiple functions as is
+designed to make the container image act like a cohesive application.
+It helps automate the provisioning of and/or connection to a domain.
+This tool is provided by the
+[sambacc project](https://github.com/samba-in-kubernetes/sambacc).
+
+<!-- TODO: link to advanced docs for samba server -->
+
+## Client
+
+The client image does not specify a default entrypoint.
+
+The project also provides a simple samba client container image that can be
+useful for testing. One does not need to install the user space samba client
+locally but rather use the container image. Example:
```

Comment:
maybe this whole paragraph should be moved up to right after the client heading?

Reply:
Yeah, I think I probably just missed moving that one. Thanks!

Reply:
I think I misunderstood the suggestion when I was not looking at it within context. I have tried to describe each image as having or not having a custom entrypoint in the first line of each subsection.  The idea was to quickly show which container images were "app-like" and which were not.  Does that make sense?